src/components/fields/password-field/password-field.js

```diff
@@ -72,16 +73,30 @@ class PasswordField extends React.Component {
     return (
       <Constraints.Horizontal constraint={this.props.horizontalConstraint}>
         <Spacings.Stack scale="xs">
-          <FieldLabel
-            title={this.props.title}
-            hint={this.props.hint}
-            description={this.props.description}
-            onInfoButtonClick={this.props.onInfoButtonClick}
-            hintIcon={this.props.hintIcon}
-            badge={this.props.badge}
-            hasRequiredIndicator={this.props.isRequired}
-            htmlFor={this.state.id}
-          />
+          <Spacings.Inline>
+            <FieldLabel
+              hint={this.props.hint}
+              title={this.props.title}
+              badge={this.props.badge}
+              htmlFor={this.state.id}
+              hintIcon={this.props.hintIcon}
+              description={this.props.description}
+              onInfoButtonClick={this.props.onInfoButtonClick}
+              hasRequiredIndicator={this.props.isRequired}
+            />
+            <FlatButton
+              icon={
+                this.state.isPasswordVisible ? <EyeCrossedIcon /> : <EyeIcon />
+              }
+              label={this.state.isPasswordVisible ? 'hide' : 'show'}
```

Comment:
You forgot to pass the message here

Reply:
...as FlatButton expects a string as `label` you have to `injectIntl` and `intl.formatMessage` here.